### PR TITLE
Chrome: GM_listValues works as expected, xkit_storage updates when values are deleted

### LINKED ISSUE #493: Chrome's GM_listValues looks in the wrong place
as noted by @nightpool in [xkit_patches.js#L228](https://github.com/new-xkit/XKit/blob/master/Extensions/xkit_patches.js#L228), chrome's `GM_listValues()` doesn't work; it looks in `window.localStorage` which is empty unless you have Retags.  This causes the check on line 228 to go awry and the full config dump is empty.

### PR DESCRIPTION
**Pull request by finagle29**
fixes #493

### CODE REVIEW

File: Chrome/bridge.js [hunk 1]
```diff
@@ -10,7 +10,7 @@
 var bridge_error = false;
 var bridge_error_object;
 var xkit_storage = {};
-var bridge_ver = "2.1";
+var bridge_ver = "2.2.1";
```
Comment (finagle29) [inline Chrome/bridge.js]: >The version in the comment on line 4 was bumped in commit b308886 but bridge_ver wasn't bumped back then so I'm just making everything consistent.

**Comment by nightpool** [comment]
would prefer a standard 7.7.2 branch

**Comment by hobinjk** [comment]
This LGTM with line https://github.com/finagle29/XKit/blob/chrome-storage-consistency/Chrome/bridge.js#L129 updated to use GM_deleteValue so that xkit_storage reflects calls to GM_deleteAllValues.

**Comment by finagle29** [comment]
Updated commit to have GM_deleteAllValues call GM_deleteValue

**Comment by nightpool** [comment]
should be updated to merge into master given our current policies on extension releases. This can be changed by the author of the PR.

**Comment by nightpool** [comment]
I'm not sure i'm super comfortable merging this without understanding exactly the implications of adding `delete xkit_storage` and some manual testing around that—maybe this would fix existing storage weirdness though?

**Pull request closed without merging**